athena-redshift/src/test/java/com/amazonaws/athena/connectors/redshift/RedshiftMetadataHandlerTest.java

```diff
@@ -330,22 +330,23 @@ public void doGetTableWithArrayColumns()
                 .forEach(expectedSchemaBuilder::addField);
         Schema expected = expectedSchemaBuilder.build();
 
-        TableName inputTableName = new TableName("testSchema", "testTable");
+        TableName inputTableName = new TableName("testSchema", "testtable");
         String[] columnNames = new String[] {"table_name"};
         String[][] tableNameValues = new String[][]{new String[] {"testTable"}};
         ResultSet resultSetName = mockResultSet(columnNames, tableNameValues, new AtomicInteger(-1));
-        String sql = "SELECT table_name, lower(table_name) FROM information_schema.tables WHERE table_name = 'testTable' or lower(table_name) = 'testTable' LIMIT 1";
+        String sql = "SELECT table_name, lower(table_name) FROM information_schema.tables WHERE (table_name = 'testtable' or lower(table_name) = 'testtable') AND table_schema = 'testSchema'";
         Mockito.when(this.connection.prepareStatement(sql).executeQuery()).thenReturn(resultSetName);
-        Mockito.when(connection.getMetaData().getColumns("testCatalog", inputTableName.getSchemaName(), inputTableName.getTableName(), null)).thenReturn(resultSet);
+        Mockito.when(connection.getMetaData().getColumns("testCatalog", inputTableName.getSchemaName(), "testTable", null)).thenReturn(resultSet);
```

Comment:
is this on purpose where you do mixed case?

Reply:
Yes, because mixed case is the resolved table name. I can extract it as a more descriptive stand alone variable, thanks for pointing out that it can cause confusion.